alchemy/src/cloudflare/worker.ts

```diff
@@ -1384,3 +1409,29 @@ async function deleteQueueConsumers(
     }),
   );
 }
+
+/**
+ * Deploy a worker to a dispatch namespace
+ * @param api CloudflareApi instance
+ * @param workerName Name of the worker script
+ * @param namespace Name of the dispatch namespace
+ */
+export async function deployWorkerToDispatchNamespace(
+  api: CloudflareApi,
+  workerName: string,
+  namespace: string,
+): Promise<void> {
+  const deployResponse = await api.put(
+    `/accounts/${api.accountId}/workers/dispatch/namespaces/${namespace}/scripts/${workerName}`,
+    {},
+  );
+
+  if (!deployResponse.ok) {
+    await handleApiError(
+      deployResponse,
+      "deploying to dispatch namespace",
+      "worker",
+      workerName,
+    );
+  }
+}
```

Comment:
delete this, you should instead be just changing the endpoint where we deploy the worker otherwise